beamerthemeNTNU.sty

```diff
@@ -85,15 +85,28 @@
 \DeclareOptionBeamer{hideframetotal}{%
 	\ntnu@displayTotalFrameNumberfalse%
 }
+
+\DeclareOptionBeamer{mathfont}[sans]{
+	
+	\ifnum\pdf@strcmp{#1}{sans}=\z@%
+		\usefonttheme{structurebold}%
```

Comment:
Why did you move the structure bold herein?

Reply:
> Why did you move the structure bold herein?

Earlier attempts of  solutions that didn't work... I will revert this when i fix the rest :)